### ISSUE TITLE
Task_2_Credit_Risk.ipynb

### ISSUE THREAD

**Issue by noormaryamqurexhi-spec** [noormaryamqurexhi-spec/DevelopersHub-Internship-Tasks#2]
Task 2: Credit Risk Prediction
import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.linear_model import LogisticRegression
from sklearn.metrics import accuracy_score

 Note: You can use a sample dataset or upload 'loan_data.csv' to GitHub
 For this example, we use the Iris data structure to show logic
import seaborn as sns
df = sns.load_dataset('iris') 
X = df.drop('species', axis=1)
y = df['species']

X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
model = LogisticRegression(max_iter=200)
model.fit(X_train, y_train)

predictions = model.predict(X_test)
print(f"Prediction Accuracy: {accuracy_score(y_test, predictions)}")
